.github/workflows/tests.yml

```diff
@@ -189,6 +189,48 @@ jobs:
           path: test-apps/e2e/**/test-results/**/trace.zip
           retention-days: 1
 
+  e2e_ee:
+    timeout-minutes: 60
+    needs: [changes, build, typescript, unit_front]
+    name: '[EE] e2e (browser: ${{ matrix.project }})'
+    runs-on: ubuntu-latest
+    env:
+      STRAPI_LICENSE: ${{ secrets.strapiLicense }}
+    strategy:
+      fail-fast: false
+      matrix:
+        project: ['chromium', 'webkit', 'firefox']
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+
+      - name: Monorepo install
+        uses: ./.github/actions/yarn-nm-install
+
+      - name: Install Playwright Browsers
+        run: npx playwright@1.38.1 install --with-deps
+
+      - name: Monorepo build
+        uses: ./.github/actions/run-build
+
+      - name: Run [EE] E2E tests
+        uses: ./.github/actions/run-e2e-tests
+        with:
+          runEE: true
+          jestOptions: --project=${{ matrix.project }}
```

Comment:
Should I do the same for CE? 🤔

Reply:
yes please, lets keep them close as possible to each other.